packages/patternfly-4/src/content/design-guidelines/usage-and-behavior/toolbar/design.md

```diff
@@ -0,0 +1,94 @@
+# Toolbar
+The toolbar contains multiple components that allow a user to manage and manipulate a data set. Data can be presented in a variety of views (table, list, card, etc.). PatternFly supports multiple toolbar variations that address a variety of use cases.
+
+## Elements
+![toolbar](img/toolbar.png)
+
+The following elements may be included in a typical toolbar.
+1. **Bulk selection (optional)**: A bulk selector may be added when this functionality is not available from the data presentation component itself. Card views and data lists may require a bulk selector. Besides allowing the user to select and deselect all items on a page, other application specific options can be added. There should also be an indeterminate state where the bulk selector notifies the user that there are some items selected on a page. When a bulk selector is included, it will always be the left-most element in the toolbar. If the view type is a list, the bulk selector should be aligned with the checkboxes in each row.
+2. **Filters**: PatternFly supports a variety of common filter types that can be used alone or in combination to construct a custom filter.
+3. **Sort (optional)**: A sort component may to added when this functionality is needed. Tables have built in sorting, but card views and data lists may require toolbar sorting.
+4. **Global actions**: Any global actions that apply to selected data items are included left-aligned following filter and sort controls. These may be exposed as buttons or collapsed into a kabob menu. No more than two items should be exposed as buttons. Button labels should be limited in length to preserve horizontal space.
+5. **Pagination**: When pagination is supported, a pagination component will be included. Pagination will always be right aligned in the toolbar.
+
+## Related components and demos
```

Comment:
I like the way you've formatted this. @stacymcauliffe should we adopt this as the standard or list inline like we've been doing? In any case I think we should be consistent.

Reply:
Agree. It's easier to read, so I think that makes up for the potential scroll-factor. I can update all of the docs to be formatted like this. 
That being said, how would you feel about moving this section to the very bottom of the documentation? Let me know. I just feel like it could be secondary information for a designer.

Reply:
One thing we could potentially do to minimize the scroll-factor is use a 2x2 table so that the Core and React lists are side by side rather than stacked. Just an idea!

Reply:
I think that we considered that earlier and decided not to do a table because it called to the mis-alignment between Core and React. But I'll defer to @stacymcauliffe on this point.